src/NewEmail/Form/index.jsx

```diff
@@ -0,0 +1,107 @@
+import {useState} from "react"
+import TextInput from "../../Components/TextInput"
+
+function Form() {
+    const [email, setEmail] = useState("")
+    const [subject, setSubject] = useState("")
+    const [textArea, setTextArea] = useState("")
+    const [send, setSend] = useState(false)
+    const [cancel, setCancel] = useState(false)
+    const [error, setError] = useState(false)
+
+    const handleCancel = () => {
+        setEmail("")
+        setSubject("")
+        setTextArea("")
+        setCancel(true)
+        setSend(false)
+    }
+
+    const handleSend = (e) => {
+        e.preventDefault()
+        if (email.trim() && subject.trim() && textArea.trim()) {
+            const newEmail = {
+                email,
+                subject,
+                body: textArea,
+                name: "MrBloggs",
+                date: new Date().toLocaleString()
```

Comment:
The date here doesn't do anything, the API just ignores it, so you might as well remove it